src/generator/AutoRest.Ruby.Azure.Tests/RspecTests/lro_spec.rb

```diff
@@ -49,8 +49,7 @@
   end
 
   it 'should retry on 202 server response in POST request' do
-    result = @lros_client.post202retry200_async(@product).value!
-    expect(result.body).to be_nil
+    result = @lros_client.post202retry200_async(@product).value!    

```

Comment:
@sarangan12 what are we expecting to come back in the response body? is there something we could check on?